app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -30,19 +31,57 @@ class HomeMenu(
         object History : Item()
         object Bookmarks : Item()
         object Quit : Item()
+        object Sync : Item()
     }
 
     val menuBuilder by lazy { BrowserMenuBuilder(menuItems) }
 
     private val hasAccountProblem get() = context.components.backgroundServices.accountManager.accountNeedsReauth()
     private val primaryTextColor =
         ThemeManager.resolveAttribute(R.attr.primaryText, context)
+    private val syncDisconnectedColor = ThemeManager.resolveAttribute(R.attr.syncDisconnected, context)
+    private val syncDisconnectedBackgroundColor =
+        ThemeManager.resolveAttribute(R.attr.syncDisconnectedBackground, context)
```

Comment:
Gave some more context over Slack, but this should be `private val syncDisconnectedBackgroundColor = context.getColorFromAttr(R.attr.syncDisconnectedBackground)`.